Cargo.toml

```diff
@@ -35,7 +35,7 @@ members = [
 ]
 
 [workspace.package]
-version = "26.0.0"
+version = "27.0.0"
```

Comment:
It is strange the diff shows these changes as they seem to have been made on master 🤔